openslides_backend/action/motion_workflow/create_update_delete.py

```diff
@@ -1,20 +1,64 @@
+from typing import Any, Dict
+
 from ...models.motion_workflow import MotionWorkflow
-from ..action import register_action_set
-from ..action_set import ActionSet
+from ...shared.exceptions import ActionException
+from ...shared.patterns import Collection, FullQualifiedId
+from ..action import register_action
 from ..default_schema import DefaultSchema
+from ..generics import CreateAction, DeleteAction, UpdateAction
 
 
-@register_action_set("motion_workflow")
-class MotionWorkflowActionSet(ActionSet):
+@register_action("motion_workflow.create")
+class MotionWorkflowCreateAction(CreateAction):
     """
-    Actions to create, update and delete motion workflows.
+    Action to create a motion workflow.
     """
 
     model = MotionWorkflow()
-    create_schema = DefaultSchema(MotionWorkflow()).get_create_schema(
+    schema = DefaultSchema(MotionWorkflow()).get_create_schema(
         properties=["name", "meeting_id"], required_properties=["name", "meeting_id"],
     )
-    update_schema = DefaultSchema(MotionWorkflow()).get_update_schema(
+
+
+@register_action("motion_workflow.update")
+class MotionWorkflowUpdateAction(UpdateAction):
+    """
+    Action to update a motion workflow.
+    """
+
+    model = MotionWorkflow()
+    schema = DefaultSchema(MotionWorkflow()).get_update_schema(
         properties=["name", "first_state_id"]
     )
-    delete_schema = DefaultSchema(MotionWorkflow()).get_delete_schema()
+
+
+@register_action("motion_workflow.delete")
+class MotionWorkflowDeleteAction(DeleteAction):
+    """
+    Action to delete a motion workflow
+    """
+
+    model = MotionWorkflow()
+    schema = DefaultSchema(MotionWorkflow()).get_delete_schema()
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        check meeting.motions_default_workflow_id and
+        meeting.motions_default_statute_amendment_workflow_id
+        """
+        workflow = self.fetch_model(
+            FullQualifiedId(Collection("motion_workflow"), instance["id"]),
+            ["meeting_id"],
+        )
+        meeting = self.fetch_model(
+            FullQualifiedId(Collection("meeting"), int(workflow["meeting_id"])),
+            [
+                "motions_default_workflow_id",
+                "motions_default_statute_amendment_workflow_id",
+            ],
+        )
+        if instance["id"] == meeting["motions_default_workflow_id"]:
```

Comment:
there is a third one: "motions_default_amendment_workflow_id"

Reply:
Done, add the third default check, too. And add a test.